torchsde/_brownian/brownian_interval.py

```diff
@@ -378,12 +370,21 @@ def __init__(self,
             shapes.append(H.shape)
             dtypes.append(H.dtype)
             devices.append(H.device)
+        if shape is not None:
+            shapes.append(shape)
+        if dtype is None:
+            if len(dtypes) == 0:
+                dtypes.append(torch.get_default_dtype())
+        else:
+            dtypes.append(dtype)
+        if device is None:
+            if len(devices) == 0:
+                # Is there no better way to get the current default device?
```

Comment:
Why not just use CPU?

Reply:
So I had it in my head that there was some way to set a non-CPU default device, but I think you're right that there isn't. I'll change this.